src/sage/data_structures/stream.py

```diff
@@ -1747,12 +1744,10 @@ def get_coefficient(self, n):
         """
         if n == 1:
             return self._ainv
-        c = self._zero
-        for k in divisors(n):
-            if k < n:
-                val = self._series[n//k]
-                if val:
-                    c += self[k] * val
+        # TODO: isn't self[k] * l and l * self[k] the same here?
```

Comment:
I am not sure how careful we need to be about noncommutative rings here. We probably should test this...